roles/load_aware_deploy_trimaran/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+- name: Configure user application monitoring
+  command:
+    oc apply -f {{ user_applications_monitor_config }}
+
+- name: Ensure user monitoring enabled
+  shell:
+    set -o pipefail;
+
+    oc -n openshift-user-workload-monitoring get pod
+      | awk 'NR > 1 { print $3}'
+  register: monitoring_enabled
+  delay: 3
+  retries: 20
+  # until all of the pods are in the running state
+  until: "'Completed' not in monitoring_enabled.stdout and 'Failed' not in monitoring_enabled.stdout and 'Pending' not in monitoring_enabled.stdout" 
+
+- name: Deploy Trimaran scheduler
+  shell:
+    set -o errexit;
+    set -o pipefail;
+    set -o nounset;
+    set -o errtrace;
+
+    export MONITORING_SECRET=$(oc get secret -n openshift-user-workload-monitoring
+      | grep  prometheus-user-workload-token
+      | head -n 1
+      | awk '{print $1 }'
+    )
+
```

Comment:
@drewrip can you add here:
```
if [[ -z "$MONITORING_SECRET" ]]; then
  echo "Coudln't not get the prometheus-user-workload-token secret, aborting"
  exit 1
fi
```

Reply:
Yes! Can do. Will include this in a PR with improvements.